packages/eslint-config/base.js

```diff
@@ -0,0 +1,28 @@
+import js from '@eslint/js'
+import eslintConfigPrettier from 'eslint-config-prettier'
+import turboPlugin from 'eslint-plugin-turbo'
+import tseslint from 'typescript-eslint'
+import onlyWarn from 'eslint-plugin-only-warn'
```

Comment:
```suggestion
import eslintJs from '@eslint/js'
import eslintTs from 'typescript-eslint'
import eslintConfigPrettier from 'eslint-config-prettier'
import eslintPluginTurbo from 'eslint-plugin-turbo'
import eslintPluginOnlyWarn from 'eslint-plugin-only-warn'
```

What do you think of this?